cloudsync/sync/state.py

```diff
@@ -107,16 +110,28 @@ def __setattr__(self, k, v):
             object.__setattr__(self, k, v)
             return
 
+        if k == "exists":
+            if v == CORRUPT and not self.is_corrupt:
+                self._saved_exists = self.exists
+                self._exists = v
+                return
+            if v != CORRUPT and self.is_corrupt:
+                self._saved_exists = self._translate_exists(v)
```

Comment:
if the caller is setting it to exists = False or exists = Missing, it should probably be allowed, right?

Reply:
I can create an "uncorrupt" method... comments are also good

Reply:
done